modules/encoders/php/base64.rb

```diff
@@ -17,6 +17,11 @@ def initialize
       'Author' => 'egypt',
       'License' => BSD_LICENSE,
       'Arch' => ARCH_PHP)
+    register_options(
+      [
+        OptString.new('Compress', [ false, 'Compress the payload with zlib' ])
```

Comment:
While testing and toggling this on and off I felt like it made sense to:
- make this a boolean value
- explicitly disable the value 
- and to copy over the reasoning for it being disabled to a comment. 
```suggestion
        OptBool.new('Compress', [ true, 'Compress the payload with zlib', false ]) # Disabled by default as it relies on having php compiled with zlib, which might not be available on come exotic setups.
```

Reply:
This was my intent, but it seems like I held `OptBool` wrong, my bad. Thanks for the fix!